src/components/note-containers/note-containers.js

```diff
@@ -3,17 +3,37 @@ import './note-containers.css';
 import TodoData from "../../service/service";
 
 export default class NoteContainers extends Component {
+    
+    todoData = new TodoData();
 
-   
+    state = {
+        notes: []
+    };
 
+    componentDidMount() {
+        this.todoData
+        .getAllNotes()
+        .then((notes) => {
+            this.setState ({
+                notes
+        });
+      });
+    }
+   
     render () {
+
+        const { notes } = this.state
+
         return (
+         
         <div className='note-containers'>
+            {notes.map(note => 
             <div className='note-container'>
-              
-            </div> 
+                <span>{note.id}</span> <br />
```

Comment:
span + br is similar as div. Use div when you need a new line. It's a block element and will take all the width of the parent element